src/SonarScanner.MSBuild.Tasks/Resources.Designer.cs

```diff
@@ -226,20 +226,11 @@ internal class Resources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Using default regular expression for detecting test projects: {0}.
+        ///   Looks up a localized string similar to Regular expression was not set in analysis config file. Project file name will not be checked..
         /// </summary>
-        internal static string IsTest_UsingDefaultRegEx {
+        internal static string IsTest_NameNotChecked {
             get {
-                return ResourceManager.GetString("IsTest_UsingDefaultRegEx", resourceCulture);
-            }
-        }
-        
-        /// <summary>
-        ///   Looks up a localized string similar to Using regular expression for detecting test projects from analysis config file: {0}.
-        /// </summary>
-        internal static string IsTest_UsingRegExFromConfig {
```

Comment:
this isn't necessary anymore?

Reply:
Hmm, it was actually logging each regex usage before. I've put it back.